[Bug] Vercel部署失败

### 📦 Environment

Vercel

### 📌 Version

v1.36.3

### 💻 Operating System

macOS

### 🌐 Browser

Chrome

### 🐛 Bug Description

一直使用Vercel部署并正常使用，没有动过配置，今天手动点击重新部署后，显示部署失败
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/cfafbedf-4011-402d-a207-15a3e869830c">

```
[12:38:32.959] Running build in Washington, D.C., USA (East) – iad1
[12:38:33.088] Cloning github.com/iparanoid/lobe-chat (Branch: main, Commit: 5d1261e)
[12:38:33.096] Skipping build cache, deployment was triggered without cache.
[12:38:34.718] Cloning completed: 1.630s
[12:38:35.406] Running "vercel build"
[12:38:35.856] Vercel CLI 39.1.3
[12:38:36.456] Running "install" command: `bun install`...
[12:38:36.504] bun install v1.1.38 (bf2f153f)
[12:38:36.522] Resolving dependencies
[12:38:52.418] Resolved, downloaded and extracted [10048]
[12:38:52.419] warn: incorrect peer dependency "eslint@8.57.1"
[12:38:52.419] 
[12:38:52.419] warn: incorrect peer dependency "stylelint@15.11.0"
[12:38:52.419] 
[12:38:52.419] warn: incorrect peer dependency "stylelint@15.11.0"
[12:38:52.420] 
[12:38:52.420] warn: incorrect peer dependency "stylelint@15.11.0"
[12:38:52.420] 
[12:38:52.420] warn: incorrect peer dependency "stylelint@15.11.0"
[12:38:52.420] 
[12:38:52.420] warn: incorrect peer dependency "@octokit/core@5.2.0"
[12:39:02.707] Saved lockfile
[12:39:02.707] 
[12:39:02.708] $ husky
[12:39:02.765] 
[12:39:02.765] + @commitlint/cli@19.6.0
[12:39:02.765] + @edge-runtime/vm@4.0.4 (v5.0.0 available)
[12:39:02.766] + @huggingface/tasks@0.12.30 (v0.13.11 available)
[12:39:02.766] + @lobehub/i18n-cli@1.20.1
[12:39:02.766] + @lobehub/lint@1.24.4
[12:39:02.766] + @lobehub/seo-cli@1.4.2
[12:39:02.766] + @next/bundle-analyzer@14.2.20 (v15.0.4 available)
[12:39:02.766] + @next/eslint-plugin-next@14.2.20 (v15.0.4 available)
[12:39:02.766] + @peculiar/webcrypto@1.5.0
[12:39:02.766] + @semantic-release/exec@6.0.3
[12:39:02.766] + @testing-library/jest-dom@6.6.3
[12:39:02.766] + @testing-library/react@16.1.0
[12:39:02.766] + @types/chroma-js@2.4.4
[12:39:02.766] + @types/debug@4.1.12
[12:39:02.766] + @types/diff@6.0.0
[12:39:02.766] + @types/fs-extra@11.0.4
[12:39:02.766] + @types/ip@1.1.3
[12:39:02.767] + @types/json-schema@7.0.15
[12:39:02.767] + @types/lodash@4.17.13
[12:39:02.767] + @types/lodash-es@4.17.12
[12:39:02.767] + @types/node@20.17.9 (v22.10.1 available)
[12:39:02.767] + @types/numeral@2.0.5
[12:39:02.767] + @types/pg@8.11.10
[12:39:02.767] + @types/react@18.3.13 (v19.0.1 available)
[12:39:02.767] + @types/react-dom@18.3.2 (v19.0.1 available)
[12:39:02.767] + @types/rtl-detect@1.0.3
[12:39:02.767] + @types/semver@7.5.8
[12:39:02.767] + @types/systemjs@6.15.1
[12:39:02.767] + @types/ua-parser-js@0.7.39
[12:39:02.767] + @types/unist@3.0.3
[12:39:02.767] + @types/uuid@10.0.0
[12:39:02.767] + @types/ws@8.5.13
[12:39:02.767] + @vitest/coverage-v8@1.2.2 (v2.1.8 available)
[12:39:02.768] + ajv-keywords@5.1.0
[12:39:02.768] + commitlint@19.6.0
[12:39:02.768] + consola@3.2.3
[12:39:02.768] + dotenv@16.4.7
[12:39:02.768] + dpdm-fast@1.0.7
[12:39:02.768] + drizzle-kit@0.29.1
[12:39:02.768] + eslint@8.57.1 (v9.16.0 available)
[12:39:02.768] + eslint-plugin-mdx@2.3.4 (v3.1.5 available)
[12:39:02.769] + eslint-plugin-unused-imports@4.0.1 (v4.1.4 available)
[12:39:02.769] + fake-indexeddb@6.0.0
[12:39:02.769] + fs-extra@11.2.0
[12:39:02.769] + glob@11.0.0
[12:39:02.769] + gray-matter@4.0.3
[12:39:02.769] + happy-dom@15.11.7
[12:39:02.769] + husky@9.1.7
[12:39:02.769] + just-diff@6.0.2
[12:39:02.769] + lint-staged@15.2.10
[12:39:02.769] + lodash@4.17.21
[12:39:02.769] + markdown-table@3.0.4
[12:39:02.769] + markdown-to-txt@2.0.1
[12:39:02.769] + node-fetch@3.3.2
[12:39:02.769] + node-gyp@10.3.1 (v11.0.0 available)
[12:39:02.769] + openapi-typescript@6.7.6 (v7.4.4 available)
[12:39:02.769] + p-map@7.0.3
[12:39:02.769] + prettier@3.4.2
[12:39:02.769] + remark-cli@11.0.0 (v12.0.1 available)
[12:39:02.769] + remark-parse@10.0.2 (v11.0.0 available)
[12:39:02.770] + semantic-release@21.1.2 (v24.2.0 available)
[12:39:02.770] + serwist@9.0.11
[12:39:02.770] + stylelint@15.11.0 (v16.11.0 available)
[12:39:02.770] + tsx@4.19.2
[12:39:02.770] + typescript@5.7.2
[12:39:02.770] + unified@11.0.5
[12:39:02.770] + unist-util-visit@5.0.0
[12:39:02.770] + vite@5.4.11 (v6.0.3 available)
[12:39:02.770] + vitest@1.2.2 (v2.1.8 available)
[12:39:02.770] + vitest-canvas-mock@0.3.3
[12:39:02.770] + @ant-design/icons@5.5.2
[12:39:02.770] + @ant-design/pro-components@2.8.2
[12:39:02.770] + @anthropic-ai/sdk@0.32.1
[12:39:02.770] + @auth/core@0.37.4
[12:39:02.770] + @aws-sdk/client-bedrock-runtime@3.706.0
[12:39:02.770] + @aws-sdk/client-s3@3.705.0
[12:39:02.771] + @aws-sdk/s3-request-presigner@3.705.0
[12:39:02.771] + @azure/core-rest-pipeline@1.16.0 (v1.18.1 available)
[12:39:02.771] + @azure/openai@1.0.0-beta.12
[12:39:02.771] + @baiducloud/qianfan@0.1.9 (v0.2.2 available)
[12:39:02.771] + @cfworker/json-schema@3.0.1 (v4.0.2 available)
[12:39:02.771] + @clerk/localizations@3.8.0
[12:39:02.771] + @clerk/nextjs@5.7.5 (v6.8.0 available)
[12:39:02.771] + @clerk/themes@2.1.53
[12:39:02.771] + @codesandbox/sandpack-react@2.19.10
[12:39:02.771] + @cyntler/react-doc-viewer@1.17.0
[12:39:02.771] + @google/generative-ai@0.21.0
[12:39:02.771] + @huggingface/inference@2.8.1
[12:39:02.771] + @icons-pack/react-simple-icons@9.6.0 (v10.2.0 available)
[12:39:02.772] + @khmyznikov/pwa-install@0.3.9 (v0.4.8 available)
[12:39:02.772] + @langchain/community@0.3.17
[12:39:02.772] + @lobehub/chat-plugin-sdk@1.32.4
[12:39:02.772] + @lobehub/chat-plugins-gateway@1.9.0
[12:39:02.772] + @lobehub/icons@1.54.0
[12:39:02.772] + @lobehub/tts@1.25.8
[12:39:02.772] + @lobehub/ui@1.153.11
[12:39:02.772] + @neondatabase/serverless@0.10.4
[12:39:02.772] + @next/third-parties@14.2.20 (v15.0.4 available)
[12:39:02.772] + @react-spring/web@9.7.5
[12:39:02.772] + @sentry/nextjs@7.120.1 (v8.42.0 available)
[12:39:02.772] + @serwist/next@9.0.11
[12:39:02.772] + @t3-oss/env-nextjs@0.11.1
[12:39:02.772] + @tanstack/react-query@5.62.3
[12:39:02.772] + @trpc/client@11.0.0-rc.660
[12:39:02.772] + @trpc/next@11.0.0-rc.660
[12:39:02.772] + @trpc/react-query@11.0.0-rc.660
[12:39:02.772] + @trpc/server@11.0.0-rc.660
[12:39:02.772] + @vercel/analytics@1.4.1
[12:39:02.772] + @vercel/edge-config@1.4.0
[12:39:02.772] + @vercel/speed-insights@1.1.0
[12:39:02.772] + ahooks@3.8.4
[12:39:02.773] + ai@3.4.33 (v4.0.13 available)
[12:39:02.773] + antd@5.22.3
[12:39:02.773] + antd-style@3.7.1
[12:39:02.773] + brotli-wasm@3.0.1
[12:39:02.773] + chroma-js@2.6.0 (v3.1.2 available)
[12:39:02.773] + dayjs@1.11.13
[12:39:02.773] + debug@4.4.0
[12:39:02.773] + dexie@3.2.7 (v4.0.10 available)
[12:39:02.773] + diff@5.2.0 (v7.0.0 available)
[12:39:02.773] + drizzle-orm@0.37.0
[12:39:02.773] + drizzle-zod@0.5.1
[12:39:02.773] + fast-deep-equal@3.1.3
[12:39:02.773] + file-type@19.6.0
[12:39:02.773] + framer-motion@11.13.1
[12:39:02.773] + gpt-tokenizer@2.7.0
[12:39:02.773] + i18next@23.16.8 (v24.0.5 available)
[12:39:02.773] + i18next-browser-languagedetector@7.2.2 (v8.0.1 available)
[12:39:02.773] + i18next-resources-to-backend@1.2.1
[12:39:02.773] + idb-keyval@6.2.1
[12:39:02.774] + immer@10.1.1
[12:39:02.774] + ip@2.0.1
[12:39:02.774] + jose@5.9.6
[12:39:02.774] + js-sha256@0.11.0
[12:39:02.774] + jsonl-parse-stringify@1.0.3
[12:39:02.774] + langchain@0.3.6
[12:39:02.774] + langfuse@3.29.1 (v3.31.1 available)
[12:39:02.774] + langfuse-core@3.29.1 (v3.31.1 available)
[12:39:02.774] + lodash-es@4.17.21
[12:39:02.774] + lucide-react@0.468.0
[12:39:02.775] + mammoth@1.8.0
[12:39:02.775] + modern-screenshot@4.5.5
[12:39:02.775] + nanoid@5.0.9
[12:39:02.775] + next@14.2.8 (v15.0.4 available)
[12:39:02.775] + next-auth@5.0.0-beta.25
[12:39:02.775] + next-mdx-remote@4.4.1 (v5.0.0 available)
[12:39:02.775] + nextjs-toploader@3.7.15
[12:39:02.775] + numeral@2.0.6
[12:39:02.775] + nuqs@1.20.0 (v2.2.3 available)
[12:39:02.776] + officeparser@4.2.0 (v5.1.1 available)
[12:39:02.776] + ollama@0.5.11
[12:39:02.776] + openai@4.76.0
[12:39:02.776] + openapi-fetch@0.9.8 (v0.13.3 available)
[12:39:02.776] + partial-json@0.1.7
[12:39:02.776] + pdf-parse@1.1.1
[12:39:02.776] + pdfjs-dist@4.4.168 (v4.9.155 available)
[12:39:02.776] + pg@8.13.1
[12:39:02.776] + pino@9.5.0
[12:39:02.776] + polished@4.3.1
[12:39:02.776] + posthog-js@1.194.5
[12:39:02.776] + pwa-install-handler@2.6.1
[12:39:02.776] + query-string@9.1.1
[12:39:02.776] + random-words@2.0.1
[12:39:02.776] + react@18.3.1 (v19.0.0 available)
[12:39:02.776] + react-confetti@6.1.0
[12:39:02.776] + react-dom@18.3.1 (v19.0.0 available)
[12:39:02.776] + react-fast-marquee@1.6.5
[12:39:02.776] + react-hotkeys-hook@4.6.1
[12:39:02.776] + react-i18next@14.0.2 (v15.1.3 available)
[12:39:02.776] + react-layout-kit@1.9.1
[12:39:02.776] + react-lazy-load@4.0.1
[12:39:02.776] + react-pdf@9.1.1
[12:39:02.776] + react-scan@0.0.40 (v0.0.41 available)
[12:39:02.776] + react-virtuoso@4.12.3
[12:39:02.776] + react-wrap-balancer@1.1.1
[12:39:02.776] + remark@14.0.3 (v15.0.1 available)
[12:39:02.777] + remark-gfm@3.0.1 (v4.0.0 available)
[12:39:02.777] + remark-html@15.0.2 (v16.0.1 available)
[12:39:02.777] + request-filtering-agent@2.0.1
[12:39:02.777] + resolve-accept-language@3.1.9
[12:39:02.777] + rtl-detect@1.1.2
[12:39:02.777] + semver@7.6.3
[12:39:02.777] + sharp@0.33.5
[12:39:02.777] + shiki@1.24.1
[12:39:02.777] + stripe@15.12.0 (v17.4.0 available)
[12:39:02.777] + superjson@2.2.2
[12:39:02.777] + svix@1.43.0
[12:39:02.777] + swr@2.2.5
[12:39:02.777] + systemjs@6.15.1
[12:39:02.777] + ts-md5@1.3.1
[12:39:02.777] + ua-parser-js@1.0.39 (v2.0.0 available)
[12:39:02.777] + unstructured-client@0.18.2
[12:39:02.777] + url-join@5.0.0
[12:39:02.777] + use-merge-value@1.2.0
[12:39:02.777] + utility-types@3.11.0
[12:39:02.777] + uuid@11.0.3
[12:39:02.777] + ws@8.18.0
[12:39:02.777] + y-protocols@1.0.6
[12:39:02.777] + y-webrtc@10.3.0
[12:39:02.778] + yaml@2.6.1
[12:39:02.778] + yjs@13.6.20
[12:39:02.778] + zod@3.23.8
[12:39:02.778] + zustand@5.0.1 (v5.0.2 available)
[12:39:02.778] + zustand-utils@1.3.2
[12:39:02.778] 
[12:39:02.778] 2724 packages installed [26.29s]
[12:39:02.778] 
[12:39:02.778] Blocked 8 postinstalls. Run `bun pm untrusted` for details.
[12:39:02.836] Detected Next.js version: 14.2.8
[12:39:02.849] Running "bun run build"
[12:39:02.854] $ next build
[12:39:03.729] Attention: Next.js now collects completely anonymous telemetry regarding usage.
[12:39:03.729] This information is used to shape Next.js' roadmap and prioritize features.
[12:39:03.729] You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
[12:39:03.729] https://nextjs.org/telemetry
[12:39:03.729] 
[12:39:03.793]   ▲ Next.js 14.2.8
[12:39:03.793] 
[12:39:03.902]    Creating an optimized production build ...
[12:39:04.384]  ○ (serwist) The service worker will not be automatically registered, please call 'window.serwist.register()' in 'componentDidMount' or 'useEffect'.
[12:39:04.390]  ✓ (serwist) Bundling the service worker script with the URL '/sw.js' and the scope '/'...
[12:42:25.553] Failed to compile.
[12:42:25.553] 
[12:42:25.553] ./node_modules/@lobehub/ui/es/DraggablePanel/index.js
[12:42:25.553] Module not found: Can't resolve 're-resizable'
[12:42:25.553] 
[12:42:25.553] https://nextjs.org/docs/messages/module-not-found
[12:42:25.553] 
[12:42:25.553] Import trace for requested module:
[12:42:25.553] __barrel_optimize__?names=DraggablePanel,DraggablePanelContainer!=!./node_modules/@lobehub/ui/es/index.js
[12:42:25.553] ./src/features/FileSidePanel/index.tsx
[12:42:25.553] 
[12:42:25.554] 
[12:42:25.554] > Build failed because of webpack errors
[12:42:25.954] error: script "build" exited with code 1
[12:42:25.962] Error: Command "bun run build" exited with 1
[12:42:29.726] 
```

### 📷 Recurrence Steps

_No response_

### 🚦 Expected Behavior

_No response_

### 📝 Additional Information

_No response_

--- a/package.json
+++ b/package.json
@@ -101,6 +101,7 @@
     ]
   },
   "dependencies": {
+    "re-resizable": "6.10.1",
     "@ant-design/icons": "^5.5.1",
     "@ant-design/pro-components": "^2.7.18",
     "@anthropic-ai/sdk": "^0.32.0",

